src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -523,18 +523,32 @@ private struct Entry {
             return false;
         }
 
-        // This is a convenience method for the internal callers that were converted from using Hashtable.
+        // This is a convenience method that were converted from using Hashtable.
         // Many were combining key doesn't exist and key exists but null value (for non-value types) checks.
         // This allows them to continue getting that behavior with minimal code delta. This is basically
         // TryGetValue without the out param
```

Comment:
This whole comment is poorly worded and unnecessary. Let's just replace the whole thing with:

``` // Method similar to TryGetValue that returns the value instead of putting it in an out param ```